mantle/platform/qemu.go

```diff
@@ -711,16 +718,10 @@ func (builder *QemuBuilder) encryptIgnitionConfig() error {
 	return nil
 }
 
-// Mount9p sets up a mount point from the host to guest.  To be replaced
-// with https://virtio-fs.gitlab.io/ once it lands everywhere.
-func (builder *QemuBuilder) Mount9p(source, destHint string, readonly bool) {
-	builder.fs9pID++
-	readonlyStr := ""
-	if readonly {
-		readonlyStr = ",readonly=on"
-	}
-	builder.Append("--fsdev", fmt.Sprintf("local,id=fs%d,path=%s,security_model=mapped%s", builder.fs9pID, source, readonlyStr))
-	builder.Append("-device", virtio(builder.architecture, "9p", fmt.Sprintf("fsdev=fs%d,mount_tag=%s", builder.fs9pID, destHint)))
+// MountVirtiofs sets up a mount point from the host to guest.
+// Note that virtiofs does not currently support read-only mounts (which is really surprising!)
```

Comment:
Indeed surprising. I'm wondering about for example `--bind-ro` is that one still read-only because when we run the `mount` unit we are mounting with the `ro` option?

Reply:
Yep exactly